src/BloomBrowserUI/BloomBrowserUI.csproj

```diff
@@ -239,6 +247,11 @@
     <Content Include="bookEdit\html\font-awesome\fonts\fontawesome-webfont.woff" />
     <Content Include="bookEdit\StyleEditor\StyleEditor.less" />
     <Content Include="bookLayout\basePage.less" />
+    <Content Include="pageChooser\blank-preview.jade" />
+    <Content Include="pageChooser\js\page-chooser.js.map" />
```

Comment:
I've removed the .map files from the .csproj. The files themselves were never checked in, but they didn't need to be here either.